### ISSUE TITLE
Certificate has expired. Endoncer: inceptionresnetv2

### ISSUE THREAD

**Issue by eguar11011** [qubvel/segmentation_models.pytorch#743]
using inceptionresnetv2 with unet architecture and with the following parameters {"encoder_name": "inceptionresnetv2", "encoder_weights": "imagenet", "in_channels": 3, "classes": 7, "activation": "softmax"} error occurs:

`<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)>`

**Comment by rikyeahh** [comment]
similar error here...

**Comment by qubvel** [comment]
Hi, as a replacement you could use encoders from timm:
https://smp.readthedocs.io/en/latest/encoders_timm.html

I will try to fix it in a while
